server.js

```diff
@@ -4019,6 +4019,44 @@ cache(function(data, match, sendBadge, request) {
   });
 }));
 
+// wordpress plugin active installs integration.
+// example: https://img.shields.io/wordpress/plugin/ai/akismet.svg for https://wordpress.org/plugins/akismet
+camp.route(/^\/wordpress\/plugin\/ai\/(.*)\.(svg|png|gif|jpg|json)$/,
+cache(function(data, match, sendBadge, request) {
+  var plugin = match[1];  // eg, `akismet`.
+  var format = match[2];
+  var apiUrl = 'http://api.wordpress.org/plugins/info/1.0/' + plugin + '.json?fields=active_installs';
+  var badgeData = getBadgeData('active installs', data);
+  request(apiUrl, function(err, res, buffer) {
+    if (err != null) {
+      badgeData.text[1] = 'inaccessible';
+      sendBadge(format, badgeData);
+      return;
+    }
+    try {
+      var total = JSON.parse(buffer).active_installs + '+';
```

Comment:
wordpress only supplies a ballpark value so no 1325 installs, but 10+ 100+ 1000+ 10K+ etc.
that's why i think there should be a + since this is also how it is displayed on wordpress.org